### PR TITLE
I hate changing logic

### PR DESCRIPTION
**Pull request by sheekap**
There are 3 tiers of prizes and I was previously awarding tier 2 to every 1 in N people and tier 3 to every one in M people where M > N.

HOWEVER 
1. This is kinda unfair
2. It's tricky to decide what N and M should be

so I've randomized the prize draw. I think. 

Also, wrote a few tests because this feels like the kind of think that should be tested. 

@kassio @goyox86 sorry for the absolute mess this PR is in (this whole project is in) but I'm doing it after hours and I'm tired 😛 

(it might actually be easier for you guys to just check the files without the change comparisons? idk, whatever suits you best and is least of an inconvenience)

### CODE REVIEW

File: app/views/prizes/show.html.erb [hunk 1]
```diff
@@ -1,12 +1,51 @@
+<!-- <div class="row prize-container">
```
Comment (kassio) [inline app/views/prizes/show.html.erb]: ✂️ I'd delete the code if you need an older version of the code you can check the log of it. 😄

Comment (sheekap) [reply app/views/prizes/show.html.erb]: Yeah sorry, meant to do that. My mistake!

File: test/models/prize_test.rb [hunk 2]
```diff
@@ -1,7 +1,46 @@
-require 'test_helper'
+require './test/test_helper'
```
Comment (kassio) [inline test/models/prize_test.rb]: Don't think you should change this path, at least not if you're using `rails test` (which is the recommended)

File: test/models/prize_test.rb [hunk 3]
```diff
@@ -1,7 +1,46 @@
...
+    10.times { prize_tiers_array << prize.prize_draw(rand(1000)) }
+
+    assert prize.number_of_bottles > 0
+    assert prize.number_of_cameras > 0
+
+    assert prize_tiers_array.include?(1)
+    assert prize_tiers_array.include?(2)
+    assert prize_tiers_array.include?(3)
+  end
+
```
Comment (kassio) [inline test/models/prize_test.rb]: ✂️ empty lines/unused comments

**Comment by kassio** [comment]
I tried to run the tests locally and got:
```console
# Running:
E

Error:
PrizeControllerTest#test_should_get_show:
ActiveRecord::Fixture::FixtureError: table "prizes" has no column named "image".
```

**Comment by kassio** [comment]
Also:
cursor_and_action_controller__exception_caught

File: db/seeds.rb [hunk 4]
```diff
@@ -7,5 +7,4 @@
 #   Character.create(name: 'Luke', movie: movies.first)
 
 Prize.destroy_all
-
-# Prize.create!
+Wallpaper.destroy_all
```
Comment (goyox86) [inline db/seeds.rb]: Make sure you don't run this after you have everything setup ;)

File: app/models/prize.rb [hunk 5]
```diff
@@ -11,13 +11,23 @@ class Prize < ApplicationRecord
...
-    elsif self.counter%50 == 0
+    elsif number < 700 && self.number_of_cameras > 0
+      decrement_cameras
       3
     else
       1
     end
   end
+
+  def decrement_cameras
```
Comment (goyox86) [inline app/models/prize.rb]: Isn't this meant to be a class method?

File: db/seeds.rb (hunk 4 above)
Comment (sheekap) [reply db/seeds.rb]: Yes I know! 😬

File: app/models/prize.rb (hunk 5 above)
Comment (sheekap) [reply app/models/prize.rb]: Possibly? I think if this were something other than CSR and I hadn't spent as much time as I already have on it, I would have completely refactored to give this the proper and correct scope. I'm on a bit of a time constraint though (and don't want to stay until 9pm again) so I'm gonna keep it as it is for now ;)

**Pull request merged**